Scripts/RemediationScripts/Remediate-ConfigureIoTHubTLSVersion.ps1

```diff
@@ -374,7 +377,11 @@ function Set-MinTLSVersionForIoTHub  {
             $logResource = @{}
             $logResource.Add("ResourceGroupName",($_.ResourceGroupName))
             $logResource.Add("ResourceName",($_.ResourceName))
-            $logResource.Add("Reason","Minimum required TLS Version configured set on IoT Hub.")    
+            if ($IsRegionSupportedforTLSEnforcement) {
+                $logResource.Add("Reason","Skipping as region is not supported for TLS enforcement")
```

Comment:
if reasonSupports the TLS enforcement then reason should be this right?
Can you confirm once if I understood something wrong?

Reply:
Have we tested these changes properly after this update?

Reply:
Yes, if region supported for TLS enforcement, then the TLS version is configured with the required TLS. Hence the Reason should be "Minimum required TLS Version configured set on IoT Hub". otherwise "Skipping as region is not supported for TLS enforcement". Updated it accordingly.